cloudant-client/src/main/java/com/cloudant/client/api/CloudantClient.java

```diff
@@ -323,6 +325,40 @@ public MetaInformation metaInformation() {
         return replicator;
     }
 
+    /**
```

Comment:
Do you think there is advantage in hiving these off to a class of their own? Similarly to what happens with View/Query etc.

Reply:
If there were lots of different endpoints I might see the point in having
```java
Scheduler sch = CloudantClient.scheduler();
sch.jobs();
sch.docs();
sch.other(); // ...
```

But it seems hardly worth the bother for two endpoints which are pretty much stateless.

_However_

One thing I have noticed is that both of these endpoints seem to share the same traits as the various other view/all_docs type of endpoints, like `offset`, `total_rows`, and presumably some mechanism for paging through.

So do you think I should try and re-use the existing mechanisms we have for doing that?

Reply:
I thought this made sense, but (from the doc at least) the `_scheduler` endpoints appear to allow `skip` and `limit`, but not `start_key` etc, which would make the key based paging we use for views useless.

Reply:
As you say it isn't really vital for a couple of methods, although I think we are up to 3 methods now. I don't know of any proposals to add more functionality to the scheduler, doing it now would potentially protect the `CloudantClient` from an overwhelming number of methods, but there isn't really a lot of boilerplate or anything to package away in a `Scheduler` class so I agree it doesn't seem worth it.